llvm/test/CodeGen/NVPTX/nvptx-debug-symbol-name.ll

```diff
@@ -0,0 +1,42 @@
+; RUN: llc -mtriple=nvptx64-nvidia-cuda -mcpu=sm_86 < %s | FileCheck %s
+
+; CHECK: .global .align 1 .b8 __func___$__Z10foo_kernelv
```

Comment:
Thanks for the test @vyom1611!  If I understand this patch correctly, it's about updating symbol names present in the debug info.  Shouldn't this test be checking that the names in the debug info have been changed?